graphql/documents/mutations/dvd.graphql

```diff
@@ -0,0 +1,36 @@
+mutation DvdCreate(
+  $name: String!,
+  $aliases: String,
+  $durationdvd: String,
+  $year: String,
+  $director: String,
+  $synopsis: String,
+  $url: String,
+  $frontimage: String,
+  $backimage: String) {
```

Comment:
These should be `$front_image` and `$back_image`. There are other places where these fields exist that should also be fixed. I won't include all of them in the review.

Reply:
I'm going to try to resolve this too.